db/migrations/0002_alter_guild_description_alter_player_guild.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2023-11-08 19:29
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='guild',
+            name='description',
+            field=models.TextField(null=True),
+        ),
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='db.guild'),
+        ),
+    ]

```

Comment:
It is a good practice for Django projects to include one migration per app per pull request. Create one migration that will have all required changes and add it to the current pull request.